src/lib.rs

```diff
@@ -61,10 +61,37 @@ mod atomic64;
 pub mod proto;
 #[cfg(all(feature = "process", target_os = "linux"))]
 pub mod process_collector;
-pub mod local;
+
+pub mod local {
+    /*!
+
+    Unsync local metrics, provides better performance.
+
+    */
+
+    // TODO: define a trait and implement the trait for metrics.
+
+    pub use super::counter::{LocalCounter, LocalCounterVec, LocalIntCounter, LocalIntCounterVec};
+    pub use super::histogram::{LocalHistogram, LocalHistogramTimer, LocalHistogramVec};
+}
+
+pub mod core {
```

Comment:
Why expose these?

Reply:
Otherwise their interface can't be seen in the documentation. I moved those referenced but commonly-won't-use things into the `core` mod.